docs/conf/portland/2023/health.rst

```diff
@@ -0,0 +1,72 @@
+
+**Vaccine Requirement**
+
+If you are showing proof of vaccination to attend in person, you will need to show that you are fully vaccinated against the COVID-19 virus. We require that you:
+
+* have completed a full round of vaccine (1 dose of a single-dose vaccine or 2 doses of a two-dose vaccine)
+* that your most recent dose was at least fourteen days before the conference start
+* that your most recent dose or booster was no more than twelve months before the conference start
+
+The following are acceptable forms of proof of full vaccination:
+
+* Government-issued or government-compliant digital vaccination verification app (such as Clear or VaxYes)
+* Physical (or digital photocopy of) COVID-19 vaccination card, record, or certificate issued by a government agency, public health agency, or other authorized vaccine provider (a photo will also suffice).
+
+The proof of vaccination record must show the name of the vaccinated individual along with one other personal identifier (such as date of birth or passport number), name of the manufacturer for vaccine received, and date(s) of vaccination. It must also include the name of the official source issuing the record (e.g., public health agency, government agency, or other authorized vaccine provider).
+
+**Negative COVID-19 Test** **Requirement**
+
+In lieu of proof of vaccination, participants may instead show proof of a negative COVID-19 test administered by a verified provider to enter the event.
+
+* Provide a negative COVID-19 test upon entry to the event for the first time.
+* The test must be a PCR, NAAT, or Antigen test taken within 48 hours of picking up your badge on-site.
+* The test may NOT be an unproctored at-home test. Tests must be administered by a verified provider, either locally in Vancouver or virtually before you travel from your home city (if time permits).
+* The test results must include your name and the date and time of the test.
+
+If you are unable to take a test before the event, we will have a rapid test on-site for you to use to gain entry.
+
+Other Testing Encouraged
+------------------------
+
+We strongly encourage attendees to test themselves for COVID before and during the conference. Rapid tests are available for sale at pharmacies in Oregon [(Oregon Government - COVID-19 Testing and Vaccine Locator)](https://govstatus.egov.com/or-oha-covid-19-testing)
+
+Additionally, we intend to provide free tests to attendees who need them, depending on availability. 
+
+Onsite health and safety measures
+---------------------------------
+
+Please expect the following measures to be in place at the event:
+
+* Proof of vaccination or negative COVID-19 test required for entry
+* Event signage indicating required health and safety protocols
+* Physically spaced audience seating in certain areas of the main ballroom
```

Comment:
Are we still planning to have additional viewing rooms with streaming from the stage that will have open windows?

Reply:
Also not a now problem IMO.

Reply:
Yea, I'm not 100% confident we can promise it, but it won't be hard to do if we have livestreaming online. Can always add it.